src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -247,6 +247,8 @@ TAP_ESC::init()
 	ConfigInfoBasicRequest   &config = packet.d.reqConfigInfoBasic;
 	memset(&config, 0, sizeof(ConfigInfoBasicRequest));
 	config.maxChannelInUse = _channels_count;
+	/* Enable closed-loop control */
+	config.controlMode = 1;
```

Comment:
the other case in this same driver is:
TAP_ESC_NO_VERIFY_CONFIG

Should we use the BOARD_ prefix or do the same for the one that's already there?